### PR TITLE
Adds link to awards page rather than dup link.

### PR DESCRIPTION
**Pull request by patcon**
This pull request has been automatically generated by prose.io.

### CODE REVIEW

**Comment by patcon** [comment]
Aside: WOW. Prose has gotten so much better.

On topic: Thought the changed link might be more useful, given that the previous link already existed closer to the beginning of the post. I went digging for the awards, as I was curious. Plus, citations are awesome.

**Comment by nschonni** [comment]
Hmm, looks like Prose blew away the metadata for you too :frowning: 
Guess it wasn't just me ... but you may want to rebase to add back the metadata

**Comment by patcon** [comment]
:+1:

**Comment by nschonni** [comment]
Posted on https://github.com/prose/prose/issues/597 as it may be related :wink:

File: _posts/2013-10-14-canadian-web-experience-toolkit.md [hunk 1]
```diff
@@ -11,6 +11,6 @@ Government websites often share many of the same basic requirements such as usab
...
 
-Today, with nearly 300 forks, and almost 100 members, [the award-winning Web Experience Toolkit](https://github.com/wet-boew/wet-boew) has been translated to more than 30 languages and is one of, if not the most active government open source projects. Members of the public and government employees from any agency are encouraged to provide feedback or submit contributions at any time. As Jackson noted, "It's all free. It's a community resources... there's no request for any funds to do any work.
+Today, with nearly 300 forks, and almost 100 members, the [award-winning](https://github.com/wet-boew/wet-boew/wiki/Accolades#wiki-Awards) Web Experience Toolkit has been translated to more than 30 languages and is one of, if not the most active government open source projects. Members of the public and government employees from any agency are encouraged to provide feedback or submit contributions at any time. As Jackson noted, "It's all free. It's a community resources... there's no request for any funds to do any work.
```
Comment (afeld) [inline _posts/2013-10-14-canadian-web-experience-toolkit.md:14]: I would still include the link to the repo:

> the [award-winning](https://github.com/wet-boew/wet-boew/wiki/Accolades#wiki-Awards) [Web Experience Toolkit](https://github.com/wet-boew/wet-boew)

Comment (nschonni) [inline _posts/2013-10-14-canadian-web-experience-toolkit.md:14]: Side-by-side links are visually confusing, and also it is linked in the previous paragraph

> Thought the changed link might be more useful, given that the previous link already existed closer to the beginning of the post

Comment (afeld) [inline _posts/2013-10-14-canadian-web-experience-toolkit.md:14]: I think I had a local minima problem :smile:

**Comment by patcon** [comment]
Thanks gents! :+1:

**Pull request merged**